src/main/java/core/selenium/webdrivers/Chrome.java

```diff
@@ -5,8 +5,18 @@
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
 
+/**
+ * This class implements the configuration of the Chrome browser.
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
 public class Chrome {
-    public WebDriver initDriver(){
+    /**
+     * This method is used for configure the Chrome browser.
+     * @return a WebDriver with configuration the Chrome browser.
```

Comment:
Please add a line to separate the description of the return.

Reply:
solved